release_notes/unreleased.md

```diff
@@ -1 +1,3 @@
 **Unreleased**
+
+* Fixed an issue in the get pcap action [PAPP-21456]
```

Comment:
Can we make the release note more specific? (e.g. Fixed wrong formatted params with query and timeframe?)

Reply:
Updated

Reply:
super, thanks!